src/double_pendulum_as_single_pendulum.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+from math import sin
+
+from pydrake.all import *
+from planar_rigid_body_visualizer import PlanarRigidBodyVisualizer
+from manipulator_dynamics import *
+
+class Controller(VectorSystem):
+    def __init__(self, rigid_body_tree, gravity):
```

Comment:
It might help a reader to start this off with a longer block comment explaining that this controller implements feedback linearization to make the double pend act like a single pend

Reply:
both done.  thanks!